pkg/pillar/types/domainmgrtypes.go

```diff
@@ -361,6 +361,7 @@ type VifInfo struct {
 	Vif     string
 	VifUsed string // Has -emu in name in Status if appropriate
 	Mac     string
+	Uplinks []string
```

Comment:
Looks like this is initialized based on netInstStatus.IfNameList, but IfNameList will change if zedrouter probing detects that a management interface is no longer working and I don't see code here reacting and updating based on such changes.

Maybe we are also missing test cases for that.

Reply:
That renaming makes sense, and then also only setting it for the switch NI so that it doesn't get abused for other NI types.

Reply:
Done (also moved under VifInfo.Vlan).
Note that during zedrouter refactoring I plan to move this function from domainmgr to zedrouter, which will make  VifInfo.Vlan unnecessary and might be removed anyway.